tests/README.md

```diff
@@ -19,12 +19,11 @@
 
 ## Framework for testing
 
-The following test frameworks and libraries will be used for new test projects **[xUnit Test Framework](https://xunit.net/), [FluentAssertions](https://fluentassertions.com/) (+ [FsUnit](https://fsprojects.github.io/FsUnit/) and [FsCheck](https://github.com/fscheck/FsCheck) when needed)**.
+The following test frameworks and libraries will be used for new test projects **[xUnit Test Framework](https://xunit.net/), ([FsUnit](https://fsprojects.github.io/FsUnit/) and [FsCheck](https://github.com/fscheck/FsCheck) when needed)**.
```

Comment:
FsUnit can also be removed btw, we don't use it anymore.

Reply:
neat, thanks, I didn't know.